src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -224,6 +224,16 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
     }
   }
 
+  /**
+   * Updates the position of the autocomplete suggestion panel to ensure that its position allows
+   * for all suggestions to be displayed within the viewport.
```

Comment:
"to ensure that its positions allows for all suggestions to be displayed within the viewport" -> "to ensure that it fits all options within the viewport"